src/modules/seventv/global-emotes.js

```diff
@@ -0,0 +1,121 @@
+import ReconnectingEventSource from 'reconnecting-eventsource';
+import watcher from '../../watcher.js';
+import settings from '../../settings.js';
+import AbstractEmotes from '../emotes/abstract-emotes.js';
+import {createEmote} from './utils.js';
+import {EmoteCategories, EmoteProviders, EmoteTypeFlags, SettingIds} from '../../constants.js';
+import {hasFlag} from '../../utils/flags.js';
+import formatMessage from '../../i18n/index.js';
+
+// 7TV global emotes can be found here: https://7tv.app/emote-sets/62cdd34e72a832540de95857
+// they seem to be clean.
+
+// TODO emotes category
```

Comment:
Looks like this is done?

Reply:
yea

Reply:
I would remove the comments above, too. The fetch below shows where they can be found.

Reply:
Done